lib/rucio/daemons/automatix/automatix.py

```diff
@@ -280,9 +212,15 @@ def run(total_workers=1, once=False, inputfile=None, sleep_time=-1):
         raise exception.DatabaseException('Database was not updated, daemon won\'t start')
 
     try:
-        sites = [s.strip() for s in config_get('automatix', 'sites').split(',')]
+        rses = [s.strip() for s in config_get('automatix', 'rses').split(',')]  # TODO use config_get_list
```

Comment:
I don't understand why this change is bundled into the commit "Use uploadclient in automatix" . how is it related to the uploadclient usage ? Why do we need this two-level 'rses', then 'sites' config parsing? Is it a new functionality to prefer the "rses" naming over "sites". If yes, why doesn't it come in a separate PR (or, at least, commit) ?

Reply:
Yes, that's new functionality that should have gone into an independent commit